content/en/tracing/llm_observability/tracing_llm_applications.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Tracing LLM Applications
+kind: guide
+---
+<div class="alert alert-info">LLM Observability is in public beta.</a></div>
+
+## Overview
+
+Your application can submit data to LLM Observability in two ways: with Datadog's [Python SDK][12], or with the [LLM Observability API][13].
+
+Each request fulfilled by your application is represented as a trace on the [LLM Observability traces list page][2] in Datadog:
+
+{{< img src="tracing/llm_observability/llm-observability-overview.png" alt="An LLM Observability trace displaying each span of a request" style="width:90%;" >}}
+
+A given trace contains spans representing each choice made by an agent or each step of a workflow. A *span* represents some unit of work that your application is performing. Spans have a start time, duration, name, tags, and attributes.
+
+Multiple spans combine to form a trace, and a *root span* is the first span in a trace.
+
+A trace can contain several kinds of spans: Agent, LLM, Workflow, and so on. The *span kind* categorizes the type of work the span is performing. Different span kinds have different parent-child relationships, and only some kinds can be the root span of a trace. For details, see [Span Kinds][4].
```

Comment:
would be great to include this as well so that new users are given a general walkthrough of what this looks like

Reply:
@barieom, this information is included on the index page, which I would argue is where it belongs -- this guide page already has some duplication against the overview just for the sake of making sense on its own, but it's mostly intended to dig into more detailed ideas that would be too granular for the overview, like span kinds and the different specific actions you can take with the SDK and API.

Do you have suggestions for how I should flesh out this part of the overview more? I do list each type and explain what it is, but maybe you would like them under headers so it's more obvious?

Reply:
I would still include the short primers on[ LLM Inference Monitoring](https://docs.google.com/document/d/1o3mp-HCtNIVQUrZLOVzjrzW_GhGAoRQvieSba9oKYQY/edit#heading=h.3z1mp82tditb), [LLM Workflow Monitoring](https://docs.google.com/document/d/1o3mp-HCtNIVQUrZLOVzjrzW_GhGAoRQvieSba9oKYQY/edit#heading=h.8tpljfhuzumx), and [LLM Agent Monitoring](https://docs.google.com/document/d/1o3mp-HCtNIVQUrZLOVzjrzW_GhGAoRQvieSba9oKYQY/edit#heading=h.y15zcq73g5w0) as bullet points before `For details, see [Span Kinds][4].` sentence since we continue to get feedback from design partners around having more examples and context as these concepts are fairly new